### PR TITLE
WIP: Fix plot_accessory

### PR DESCRIPTION
**Pull request by JDWarner**
This work in progress fixes a syntax error preventing this file from executing. Additional PEP8 fixes throughout the file.

There are a few functions which reference a variable `plotdata` which is not passed. I commented out those lines to call them out; advice appreciated about how to fix the issue (adding a `plotdata` arg or changing the name to the proper arg).

### CODE REVIEW

File: trendvis/plot_accessory.py [hunk 1]
```diff
@@ -134,12 +135,14 @@ def shift_list(shift_by, num):
...
 
+    return shift_degrees
+
 
 def reordering(reorder, ratios, ticks, labels, axis_shift, to_twin, reverse_ax,
                bar_location, limits):
     """
     """
-    plotdata = reorder_bysort(reorder, plotdata)
+    # plotdata = reorder_bysort(reorder, plotdata)
```
Comment (JDWarner) [inline trendvis/plot_accessory.py]: `plotdata` not defined in `reordering`

File: trendvis/plot_accessory.py [hunk 2]
```diff
@@ -186,7 +188,7 @@ def reorder_bysort(reorder, to_sort):
 
     rlen = len(reorder)
     re_sort = [x for (y, x) in sorted(zip(reorder, to_sort[:rlen]))]
-    re_sort.extend(plotdata[rlen:])
+    # re_sort.extend(plotdata[rlen:])
```
Comment (JDWarner) [inline trendvis/plot_accessory.py]: `plotdata` not defined in `reorder_bysort` - is this supposed to be `re_sort` similar to how `reorder_byindex` works below?

**Comment by mscross** [comment]
`plotdata` should be an argument in `reordering()`; the `plotdata` in `re_sort.extend(plotdata[rlen:])` should probably be `to_sort`

**Comment by JDWarner** [comment]
Would `plotdata` be the first argument in `reordering`?

**Comment by mscross** [comment]
Yes

**Comment by JDWarner** [comment]
Done!

**Pull request merged**